test/Interop/Cxx/function/template-irgen-windows.swift

```diff
@@ -0,0 +1,40 @@
+// RUN: %target-swift-emit-ir %s -I %S/Inputs -enable-cxx-interop | %FileCheck %s
+// REQUIRES: OS=windows-msvc
+
+import FunctionTemplates
+
+// CHECK-LABEL: define {{.*}}i32 @"$s4main20testPassThroughConst1xs5Int32VAE_tF"(i32 %0)
```

Comment:
Just an idea for consideration, I typically use `{{foo|bar}}` in tests to make them work on all the platforms, e.g. https://github.com/apple/swift/blob/main/test/Interop/Cxx/static/static-var-irgen.swift#L11

Reply:
I've used that before. In this test, every two or three check lines would have to have `{{foo|bar}}` in them which makes the test super hard to read IMO. If you'd like, I can consolidate these into one test and use `{{foo|bar}}` but, I'd rather keep it as two tests.

Reply:
It would be really great if we could cross-compile the stdlib. Then we could just have multiple run lines (the same way #30630 does it).

Reply:
I personally do single test for all platforms because in the future, when I break the compiler, I want to see all the tests that I have to fix fail. And I'm developing on Linux, so in that case I wouldn't catch the failing windows test, and it's only the presubmit check that would fail. And that's much slower feedback cycle.

But yes, I often have tests with `{{foo|bar}}` on almost every check line and I understand it's making the test less understandable when a reader wants to understand what's exactly happening on a particular platform.

Re multiple RUN lines, yeah I do the same with my tests when I'm debugging them, but it felt wrong to submit that.

Reply:
Fair enough. Done.